Maestro/subscriptions.json

```diff
@@ -689,6 +689,23 @@
         }
       }
     },
+    // Automate merging aspnetcore release/6.0-rc & release/6.0-ga branches back to release/6.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/aspnetcore/blob/release/6.0-//**/*"
```

Comment:
What is the -ga branch? aspnetcore specific? The rest of the stack won't have one. release/6.0 is the GA/servicing branch.

Also, I'm not sure that this will do what you want. I think this will only merge branches that are 6.0-, not 6.0-*

Reply:
Oh, I thought there was going to be a separate branch for GA as well. Does it look right now?